### PR TITLE
Oberon 2

### PR DESCRIPTION
**Pull request by CrabbytheCrab**
## About The Pull Request
I fucked up big time and had to port the code over. It has a weapon this time being a buffed version of mockery(sprites are needed for it)

## Why It's Good For The Game
Waiter waiter more boss fights please!

## Changelog
:cl:
add: Added Oberon
/:cl:

### CODE REVIEW

**Comment by CrabbytheCrab** [comment]
I gave it an experimental shitty mimicry mode

**Comment by CrabbytheCrab** [comment]
I'll probably remove it though

File: code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm [hunk 1]
```diff
@@ -264,24 +272,108 @@
...
+				heal_amount *= 2
+			adjustBruteLoss(-heal_amount)
+			last_heal_time = world.time
+
 	if(next_transform && (world.time > next_transform))
 		next_stage()
 	if(current_stage == 2) // Egg
 		var/obj/effect/temp_visual/decoy/D = new /obj/effect/temp_visual/decoy(get_turf(src), src)
 		animate(D, alpha = 0, transform = matrix()*1.2, time = 7)
+	if(disguise)
```
Comment (Coxswain-Navigator) [inline code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm:300]: Check if shelled is true as opposed to if disguise is true, as disguise can turn into a null value under certain conditions

**Review by Coxswain-Navigator — APPROVED** [review]
I want this merged; I can give it a facelift later if needed.

**Pull request merged**